### ISSUE TITLE
MWException from line 335 of /.../w/includes/MagicWord.php: Error: invalid magic word 'coordinates'

### ISSUE THREAD

**Issue by kghbln** [JeroenDeDauw/Maps#263]
**Setup**

- MediaWiki 	1.27.1 (a52d35d) 
- PHP 	5.6.26-0+deb8u1 (apache2handler)
- Semantic MediaWiki	2.5.0-alpha (320f99b)
- Maps 4.0.0-alpha https://github.com/JeroenDeDauw/Maps/commit/9692ac00e39fb08f5a045e67ed1deb5085644de9

**Issue**

[99c9c911abb39d317d2d6126] /wiki/Accueil MWException from line 335 of /.../w/includes/MagicWord.php: Error: invalid magic word 'coordinates'

**Backtrace**
```
#0 /.../w/includes/MagicWord.php(260): MagicWord->load(string)
#1 /.../w/includes/parser/Parser.php(5232): MagicWord::get(string)
#2 /.../w/extensions/Validator/src/legacy/ParserHook.php(210): Parser->setFunctionHook(string, array, integer)
#3 /.../w/extensions/Maps/Maps.php(112): ParserHook->init(Parser)
#4 [internal function]: {closure}(Parser)
#5 /.../w/includes/Hooks.php(195): call_user_func_array(Closure, array)
#6 /.../w/includes/parser/Parser.php(335): Hooks::run(string, array)
#7 [internal function]: Parser->firstCallInit()
#8 /.../w/includes/StubObject.php(105): call_user_func_array(array, array)
#9 /.../w/includes/StubObject.php(129): StubObject->_call(string, array)
#10 /.../w/includes/cache/MessageCache.php(1051): StubObject->__call(string, array)
#11 /.../w/includes/cache/MessageCache.php(1051): StubObject->firstCallInit()
#12 /.../w/includes/cache/MessageCache.php(1028): MessageCache->getParser()
#13 /.../w/includes/Message.php(1162): MessageCache->transform(string, boolean, Language, Title)
#14 /.../w/includes/Message.php(829): Message->transformText(string)
#15 /.../w/includes/Message.php(892): Message->toString()
#16 /.../w/includes/OutputPage.php(927): Message->text()
#17 /.../w/includes/OutputPage.php(975): OutputPage->setHTMLTitle(Message)
#18 /.../w/includes/page/Article.php(504): OutputPage->setPageTitle(string)
#19 /.../w/includes/actions/ViewAction.php(44): Article->view()
#20 /.../w/includes/MediaWiki.php(503): ViewAction->show()
#21 /.../w/includes/MediaWiki.php(288): MediaWiki->performAction(Article, Title)
#22 /.../w/includes/MediaWiki.php(745): MediaWiki->performRequest()
#23 /.../w/includes/MediaWiki.php(519): MediaWiki->main()
#24 /.../w/index.php(43): MediaWiki->run()
#25 {main}
```

For the time being I pulled Maps master and run v3.8.2

**Comment by JeroenDeDauw** [comment]
I suppose this got broken when the initialization code got moved around. Perhaps the i18n code needs to be registered right away and cannot be in wgExtensionFunctions? @Nikerabbit 

@kghbln can you try and see if this change fixes the issue https://github.com/JeroenDeDauw/Maps/pull/264? Not possible for me to test without a wiki

**Comment by kghbln** [comment]
> @kghbln can you try and see if this change fixes the issue #264?

Affirmative, this fixes the issue.

>  Perhaps the i18n code needs to be registered right away and cannot be in wgExtensionFunctions?

This is obviously the case given the result of my field test.
